### PR TITLE
Add several spelling corrections + refine preexisting 

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -22135,12 +22158,23 @@ tetrahedran->tetrahedron
...
 tetxture->texture
+tetxtured->textured
+tetxtures->textures
+tetxturing->texturing
 texchnically->technically
 texline->textline
 textfrme->textframe
 texual->textual
 texually->textually
+texural->textual
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: texural->textural

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -22135,12 +22158,23 @@ tetrahedran->tetrahedron
...
+texurally->textually
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: texurally->texturally

**Comment by luzpaz** [comment]
@c72578 your raise a fair point that `textural` can also be an alternative to `textual` typos. Will need to revise. I may just split that in to a different PR.

**Comment by luzpaz** [comment]
All 'textual' typos have been split in to a separate PR that I will submit soon. Please merge this PR as is.

**Pull request merged**